test/e2e/commands/xmlupload/test_resources.py

```diff
@@ -2,63 +2,179 @@
+        iiif_uri_val = next(g.objects(iiif_uri, KNORA_API.hasStillImageFileValue))
+        assert next(g.objects(iiif_uri_val, RDF.type)) == KNORA_API.StillImageExternalFileValue
+        assert next(g.objects(iiif_uri_val, KNORA_API.hasPermissions)) == OPEN_PERMISSIONS
+        assert next(g.objects(iiif_uri_val, KNORA_API.hasAuthorship)) == Literal("Cavanagh, Annie")
+        assert next(g.objects(iiif_uri_val, KNORA_API.hasCopyrightHolder)) == Literal("Wellcome Collection")
+        assert next(g.objects(iiif_uri_val, KNORA_API.hasLicense)) == URIRef("http://rdfh.ch/licenses/cc-by-nc-4.0")
 
     @pytest.mark.usefixtures("_xmlupload")
     def test_audio(self, onto_iri, auth_header, project_iri, creds):
         cls_iri_str = f"{onto_iri}TestAudioRepresentation"
-        resources = _util_request_resources_by_class(cls_iri_str, auth_header, project_iri, creds)
-        assert len(resources) != 0
+        g = _util_request_resources_by_class(cls_iri_str, auth_header, project_iri, creds)
+        resource_iris = list(g.subjects(RDF.type, URIRef(cls_iri_str)))
+        assert next(g.objects(predicate=RDFS.label)) == Literal("audio")
+        expected_number = 1
+        assert len(resource_iris) == expected_number
 
     @pytest.mark.usefixtures("_xmlupload")
     def test_video(self, onto_iri, auth_header, project_iri, creds):
         cls_iri_str = f"{onto_iri}TestMovingImageRepresentation"
-        resources = _util_request_resources_by_class(cls_iri_str, auth_header, project_iri, creds)
-        assert len(resources) != 0
+        g = _util_request_resources_by_class(cls_iri_str, auth_header, project_iri, creds)
+        resource_iris = list(g.subjects(RDF.type, URIRef(cls_iri_str)))
+        assert next(g.objects(predicate=RDFS.label)) == Literal("video")
+        expected_number = 1
+        assert len(resource_iris) == expected_number
 
 
 class TestDspResources:
     @pytest.mark.usefixtures("_xmlupload")
     def test_region(self, auth_header, project_iri, creds):
         cls_iri_str = f"{KNORA_API_STR}Region"
-        resources = _util_request_resources_by_class(cls_iri_str, auth_header, project_iri, creds)
-        assert len(resources) != 0
+        g = _util_request_resources_by_class(cls_iri_str, auth_header, project_iri, creds)
+        resource_iris = list(g.subjects(RDF.type, URIRef(cls_iri_str)))
+        expected_number = 1
+        assert len(resource_iris) == expected_number
+        res_iri = resource_iris.pop(0)
+        res_triples = list(g.triples((res_iri, None, None)))
+        assert len(list(g.objects(res_iri, KNORA_API.hasColor))) == 1
+        assert len(list(g.objects(res_iri, KNORA_API.isRegionOfValue))) == 1
+        assert len(list(g.objects(res_iri, KNORA_API.hasGeometry))) == 1
+        assert len(list(g.objects(res_iri, KNORA_API.hasComment))) == 1
+        expected_number_of_triples = NUMBER_OF_RESOURCE_TRIPLES_WITHOUT_VALUES + 4
```

Comment:
could you add some clarity regarding the 4? Perhaps an inline comment, or define a variable with a telling name for the 4